packages/xstate-react/test/useActor.test.tsx

```diff
@@ -140,8 +157,9 @@ describe('useActor', () => {
 
     const Test = () => {
       const [state] = useMachine(machine);
+      const { actorRef } = state.context;
 
-      return <ChildTest actor={state.context.actorRef} />;
+      return actorRef ? <ChildTest actor={actorRef} /> : null;
```

Comment:
It seems that last time I've commented on the wrong line - although the comment was still sound for that other place - so gonna just link to that older comment:
https://github.com/davidkpiano/xstate/pull/1622#discussion_r524386007

```suggestion
      return actorRef ? <ChildTest actor={actorRef} /> : null;
```
```suggestion
      return <ChildTest actor={actorRef!} />;
```